docs/user-guide/cli-swreqplugins.md

```diff
@@ -5,10 +5,10 @@ Before installing a Zowe&trade; CLI plug-in, meet the software requirements to r
 :::info Required role: systems administrator
 :::
 
-| Plug-in | Requirements |
+| Plug-in | Requirements!LLL |
```

Comment:
What is `!LLL`?

Reply:
Was running into errors and added random characters to test local builds. Forgot to remove them lol